metacat-metadata-mysql/src/main/java/com/netflix/metacat/metadata/mysql/MySqlTagService.java

```diff
@@ -48,11 +50,26 @@
 
 /**
  * Tag service implementation.
+ *
+ * @author amajumdar
+ * @author zhenl
  */
 @Slf4j
 @SuppressFBWarnings
 @Transactional("metadataTxManager")
 public class MySqlTagService implements TagService {
+    /**
+     * QUALIFIEDNAME TYPE REGEX MAP.
+     */
+    private static final Map<QualifiedName.Type, String>
```

Comment:
Can we add this to the enum?

Reply:
why add this mysql regex query to the enum?

Reply:
Enum will carry extra information that can be used by any process.

Reply:
Sure.